index.html

```diff
@@ -233,6 +233,59 @@
       </p>
       <section>
         <h3>
+          Installing via automated prompting
+        </h3>
+        <p>
+          User agents that support an <a>automated install prompt</a> MUST run
```

Comment:
I'm not a huge fan of MAY ... MUST constructions. Though we need to make it clear that user agent MAY support automated install prompts elsewhere. It's part of the big rewrite this whole install section needs.